package.json

```diff
@@ -63,6 +63,7 @@
     "react-intl": "5.20.10",
     "react-is": "16.13.1",
     "react-overlays": "1.2.0",
+    "react-popper": "2.2.5",
```

Comment:
@AshishDhama did we end up using popper?

Reply:
@Willyfrog Nevy is using it for task list, Didn’t had discussions with her on this.